canteenAutomation/settings.py

```diff
@@ -93,6 +97,15 @@
     }
 }
 
+# Channels
+CHANNEL_LAYERS = {
+    "default": {"BACKEND": "channels.layers.InMemoryChannelLayer"},
```

Comment:
Default layer is an in-memory layer, so no need for any extra server while developing.